index.js

```diff
@@ -280,25 +292,44 @@ async function fetchExportFileById(type, envId, id) {
 
 async function fetchAllExportFiles(type, envId, page = 1) {
   const templates = await SF.readExportFiles(type, envId, page);
+
   if (templates.length == 0) {
     if (page == 1) {
-      consola.error(`No export files found in firm ${firmId}`);
+      consola.error("No export files found in firm");
     }
     return;
   }
+
   templates.forEach(async (template) => {
-    fetchExportFileById(type, envId, template.id);
+    const saved = ExportFile.save(type, envId, template.id);
+
+    if (saved) {
+      consola.success(`Export file "${template.name}" imported`);
+    }
   });
   fetchAllExportFiles(type, envId, page + 1);
 }
 
-async function fetchExistingExportFiles(firmId) {
+async function fetchExistingExportFiles(type, envId) {
   const templates = fsUtils.getAllTemplatesOfAType("exportFile");
-  if (!templates) return;
+  if (!templates) {
+    consola.warn("No export files found");
+    return;
+  }
+
   templates.forEach(async (name) => {
     const templateConfig = fsUtils.readConfig("exportFile", name);
-    if (!templateConfig || !templateConfig.id[firmId]) return;
-    await fetchExportFileById(firmId, templateConfig.id[firmId]);
+    let templateId = 
+      type == "firm"
+        ? templateConfig?.id?.[envId]
+        : templateConfig?.partner_id?.[envId]
```

Comment:
I know this is how it's implemented already, but I would love to see this moved (from everywhere) into a helper function

```js
function getTemplateId(type, envId, templateConfig) {
  switch (type) {
    case "firm":
      return templateConfig?.id?.[envId];
      break;
    case "partner":
      return templateConfig?.partner_id?.[envId];
      break;
    default:
      throw "Invalid template type: ${type}";
      break;
}
```
And we store in one of the utils files, where we have other config helpers.
What do you think? (Also, not completly sure it works exactly as it is, I just wrote it here while doing the review :slightly_smiling_face: )

we do later:
```js
let templateId = fsUtils.getTemplateId(type, envId, templateConfig);
```

Reply:
Implemented it as requested